web/locales/common-voice/en/pages/contribute/write.ftl

```diff
@@ -151,11 +151,13 @@ write-contribute-menu-tooltip = Add and Review Sentences, Add Questions, Transcr
 add-sentences-menu-item-tooltip = Add sentences in your language
```

Comment:
I think here and in the subsequent ones, "in your language" is probably redundant.